src/test/java/SalesforceModules/QuoteUtil.java

```diff
@@ -74,7 +78,25 @@ public static void selectOpportunity(String oppurt, String logStep) throws Inter
             KeywordUtil.click(By.xpath(xpath), "opportunity selected");
         }
     }
+    public static void selectOpportunityForLeads(String oppurt, String logStep) throws InterruptedException {
```

Comment:
Please add comments for method description and parameters

Reply:
Updated as per comments